public/locales/en/translation.json

```diff
@@ -3,9 +3,67 @@
     "about": "About",
     "acknowledgements": "Acknowledgements"
   },
-  "thankyoupage": {
+  "thankYouDialog": {
     "title": "Acknowledgements",
-    "contenttitle": "This project was developed by:",
-    "names": ["Yuval Bar Levi", "Miki Shtenger", "Yael Louria", "Meital Lazorovich"]
+    "contentTitle": "This project was developed by:",
+    "names": [
+      "Yuval Bar Levi",
+      "Miki Stanger",
+      "Meital Lazarovich",
+      "Atalya Alon",
+      "Eyal Levy",
+      "Carmel Paradis",
+      "Daniel Shely",
+      "Dror Reshef",
+      "Sergey Bekker",
+      "Eyal Shaar",
+      "Yael Luria",
+      "Yogev Boaron Ben-Har",
+      "Tal Licht",
+      "Ron Fybish",
+      "Adele Angel",
+      "Moran Zivoni"
+    ]
+  },
+  "aboutDialog": {
+    "title": "About",
+    "content1": "ANYWAY’s next challenge is to form an automatic generator of infographics to empower and serve journalists, bloggers, public opinion leaders, community leaders etc. in the era of data journalism.",
+    "content2": "The generated infographics will enhance reporting and news writing with the use of statistics. Each infographic will be created for a real-time road accident related news flash and will provide a deeper insight into the story based on historical data.",
+    "content3": "This, we believe, will increase both the quantity and quality of articles dealing with road accidents, and will result in raising public awareness and creating pressure on decision makers to initiate infrastructure improvements in light of",
+    "linkText": "Vision Zero.",
+    "linkUrl": "https://en.wikipedia.org/wiki/Vision_Zero"
+  },
+  "filterBar": {
+    "yearChoices": ["Past Year", "Last 3 Years", "Last 5 Years", "Last 8 Years"]
```

Comment:
What will happens if the translation file failed to load?
I rather have a fallback option.
You might need to use object instead of array for that.

Also, if it makes it easier - you may use `ts` instead of json (although it is not mandatory)